From 8c9bbab3311c653e96af2bb2918636e3dffc35e3 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Tue, 10 Sep 2019 12:02:23 +0800
Subject: [PATCH] dt-bindings: axp20x-adc: allow to use TS pin as GPADC

The AXP series PMICs allow to use the TS pin as GPADC rather than the
battery temperature sensor.

Add the property to enable this mode.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 .../devicetree/bindings/iio/adc/x-powers,axp209-adc.yaml     | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Documentation/devicetree/bindings/iio/adc/x-powers,axp209-adc.yaml b/Documentation/devicetree/bindings/iio/adc/x-powers,axp209-adc.yaml
index d6d3d8590171..318f6238c52e 100644
--- a/Documentation/devicetree/bindings/iio/adc/x-powers,axp209-adc.yaml
+++ b/Documentation/devicetree/bindings/iio/adc/x-powers,axp209-adc.yaml
@@ -61,6 +61,11 @@ properties:
   "#io-channel-cells":
     const: 1
 
+  x-powers,ts-as-gpadc:
+    description: |
+      Boolean. Defines whether the TS pin should be used
+      as GPADC rather than battery temperature sensor pin.
+
 additionalProperties: false
 
 examples:
-- 
2.39.2

